docs/en/engines/table-engines/integrations/s3.md

```diff
@@ -168,13 +170,30 @@ The following settings can be specified in configuration file for given endpoint
         <!-- <use_environment_credentials>false</use_environment_credentials> -->
         <!-- <use_insecure_imds_request>false</use_insecure_imds_request> -->
         <!-- <expiration_window_seconds>120</expiration_window_seconds> -->
+        <!-- <no_sign_request>false</no_sign_request> -->
         <!-- <header>Authorization: Bearer SOME-TOKEN</header> -->
         <!-- <server_side_encryption_customer_key_base64>BASE64-ENCODED-KEY</server_side_encryption_customer_key_base64> -->
         <!-- <max_single_read_retries>4</max_single_read_retries> -->
     </endpoint-name>
 </s3>
 ```
 
+## Accessing public buckets
+
+ClickHouse tries to fetch credentials from many different types of sources.
+Sometimes, it can produce problems when accessing some buckets that are public causing the client to return `403` error code.
+This issue can be avoided by using `NOSIGN` keyword, forcing the client to ignore all the credentials, and not sign the requests.
+
+``` sql
+SELECT *
+FROM s3(
```

Comment:
as this is a section about table engines, then add an example with table engine instead of table function?